MAUI/Autocomplete/Selection.md

```diff
@@ -78,12 +78,76 @@ The following gif image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete with single selection mode](Images/Selection/SingleSelection.png)
 
+## Multiple selection
+
+The AutoComplete allows user to select multiple values by beginning to enter the input and selecting items from the drop-down list. The multi-select AutoComplete mode can be enabled by setting the `SelectionMode` property as `Multiple`. 
+
+Selected items will be displayed with a  token representation and each tokenized items can be removed by clicking their close button. The selected items can be retrieved from the `SelectedItems` property.
```

Comment:
Selected items will be displayed with a token representation, and each tokenized item can be removed by clicking the close button. The selected items can be retrieved from the `SelectedItems` property.